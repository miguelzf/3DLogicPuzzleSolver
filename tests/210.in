G - - Y G
- - C - -
- - - - -
- X X X -
- - - - -

- - - - -
- - - C -
- - B - -
- - Y - X
- - - - X

- - B - -
- - X - -
- - X - -
- - - - -
- - - - -
