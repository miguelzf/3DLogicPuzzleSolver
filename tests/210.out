
Solution:
G  y  y  Y  G  
g  y  C  c  c  
g  y  y  y  y  
g  X  X  X  y  
g  y  y  y  y  
=============
g  y  b  b  b  ||  b  b  B  c  g  
g  y  b  C  c  ||  c  c  X  c  g  
g  y  B  g  g  ||  g  c  X  c  g  
g  y  Y  g  X  ||  g  c  c  c  g  
g  g  g  g  X  ||  g  g  g  g  g  
